### DISCUSSION TITLE
When creating 6502 support from tmLanguage file, how to load

### DISCUSSION THREAD

**Discussion by cpgb85** [ajaxorg/ace#5016] [Q&A]
### Describe the issue

There is no clear cut way on how to load these files. The documentation tells you how to create the file,but if you are using a CDN, how are these new files implemented?

The attached imageshows the errors i get while trying to use this.  Also, the documentation on the main site mentions nothingabout worker files.

<img width="711" alt="image" src="https://user-images.githubusercontent.com/21691341/208355382-9204b6d5-1930-470a-bf7e-b4a429c48484.png">

### Links

https://ace.c9.io/#nav=higlighter

**Comment by InspiredGuy** [comment]
Hey @cpgb85 , from the error I see on the screenshot there was a [similar issue](https://github.com/ajaxorg/ace/issues/4597) people were facing, and it was solved by setting a base path to ace config (it might be a path to cdn, a path to node modules etc. depending on your setup), by using `ace.config.set('basePath', 'path_to_ace_script');`.
The other option you could try would be using `editor.getSession().setMode(instanceOfYourCustomMode)` - see [API docs](https://ajaxorg.github.io/ace-api-docs/classes/Ace.EditSession.html#setMode).
If that doesn't help, please provide a code snippet of how you are trying to load the custom language mode into ace - that would help a lot to understand the issue at hand.
